tests/framework/session/test_session.py

```diff
@@ -504,32 +504,18 @@ def test_log_error(self, fake_project, mock_package_name):
         )
 
     @pytest.mark.usefixtures("mock_settings")
-    def test_get_current_session(self, fake_project, mock_package_name):
-        assert get_current_session(silent=True) is None  # no sessions yet
-
-        pattern = "There is no active Kedro session"
-        with pytest.raises(RuntimeError, match=pattern):
-            get_current_session()
-
+    def test_active_session(self, fake_project, mock_package_name):
```

Comment:
`test_nested_sessions` would be a more explicit name for me, but it's entirely optional.